### PR TITLE
Allow specifying the output directory for a plugin

### PR DESCRIPTION
**Pull request by edenman**
My plugin injects code into the already-generated java classes, and thus needs to have outputDir="java".

### CODE REVIEW

File: src/main/groovy/com/google/protobuf/gradle/GenerateProtoTask.groovy [hunk 1]
```diff
@@ -236,6 +237,20 @@ public class GenerateProtoTask extends DefaultTask {
     public String getName() {
       return name
     }
+
+    public setOutputDir(String outputDir) {
```
Comment (zhangkun83) [inline src/main/groovy/com/google/protobuf/gradle/GenerateProtoTask.groovy]: Maybe `setOutputSubDir()`, which reminds the user that this is always relative to the base output dir.
Also please add docs for this method.

Comment (edenman) [inline src/main/groovy/com/google/protobuf/gradle/GenerateProtoTask.groovy]: Done.

**Comment by edenman** [comment]
Thanks @zhangkun83!  Is there somewhere I can subscribe to get notified when this gets released?

**Comment by zhangkun83** [comment]
It will be included in the next release, which will be tagged. Watching or starring this project should give you notifications on new releases, but I am not 100% sure.

**Comment by zhangkun83** [comment]
@edenman FYI 0.7.0 has been released

**Comment by edenman** [comment]
@zhangkun83 awesome, thanks!

**Pull request merged**